src/main/webapp/about.jsp

```diff
@@ -66,5 +66,29 @@
       </p>
     </div>
   </div>
+<div id="container">
+    <div
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+
+      <h1>About the Team</h1>
+
+      <ul>
+        <li><strong>Heng Sun (PA)</strong> Our supereme and righteous leader(PA). 
+        </li>
+        <li><strong>Sherry Yang</strong> </li>
+        <li><strong>Dean Chiu</strong> </li>
+        <li><strong>Naomi McCracken</strong> Naomi is a third year Mathematics 
+            and Computer Science (undergraduate) student at University of 
+            California San Diego. Naomi is especially interested in working in 
+            backend development, and she especially enjoys learning about about 
+            algorithms and data structures. Her favorite sorting algorithm is 
+            Bogosort, particularly the version that produces permutations of 
+            input until the sorted version is produced. Naomi is excited to be
+            a part of the CodeU program and learn a lot alongside her teammates.
+            </li>
```

Comment:
Indentation.  I see that you're trying to match the formatting of the block above, but it's unfortunate because the block above has inconsistent indentation....

Here's my recommendation.

- Indent the block of text 2 spaces beyond <li>.
- Have end </li> after a block of text on a separate line aligned with the opening <li>.

It'd be great if you could fix the About CodeU Chat App section to be consistent as well.